tests/integration/strategy_tests.cpp

```diff
@@ -1314,8 +1236,8 @@ TEST(HeuristicTestList, EdgeCases_0)
   auto h_fun = umpire::strategy::heuristic_percent_releasable_list(0);
 
   auto alloc = rm.makeAllocator<umpire::strategy::DynamicPoolList>(
-      "host_dyn_pool__list_h_0", rm.getAllocator("HOST"),
```

Comment:
Since you added the new pool test file should these be moved over there?

Reply:
This is done.